src-web/reducers/ResourceTableReducer.js

```diff
@@ -18,18 +18,32 @@
 
 // Intital State of Redux store
 const initState = {
-    resourceTableError:''
+    resourceTableError:'',
+    componentView : {}
```

Comment:
By adding this to the ResourceTableReducer, you have changed the value that the Reducer provides.  It no longer provides just the ResourceTable information.  We should change the name of the reducer or move it to a separate reducer.